package.json

```diff
@@ -133,6 +133,7 @@
     "core-js": "^2.6.5",
     "flexboxgrid": "^6.3.1",
     "formik": "^2.0.6",
+    "lodash": "^4.17.15",
```

Comment:
if we are introducing lodash only for this utility of uniqueId across the application, then lets write our own logic to generate uniqueId or just copy that particular utility method. @vinodloha your thoughts?

Reply:
@bhardwajamit0890  Lodash has modularised their code a lot. 
Just use: https://www.npmjs.com/package/lodash.uniqueid
I know what you have done will also result in a tree-shaken code, but still ;)